packages/query/src/rate-limiter/hiro-rate-limiter.ts

```diff
@@ -25,6 +27,8 @@ export function useHiroApiRateLimiter(): PQueue {
       return hiroStacksMainnetApiLimiter;
     case 'testnet':
       return hiroStacksTestnetApiLimiter;
+    default:
+      return hiroStacksMainnetApiLimiter;
```

Comment:
Why does it need a default? Isn't this dead code if `currentNetwork.mode` = `testnet | mainnet`

Reply:
hm, I feel like there was type err bc of it...
removed